corehq/apps/hqcase/esaccessors.py

```diff
@@ -1,29 +0,0 @@
-from corehq.apps.es.cases import CaseES
-
-
-def scroll_case_ids_by_domain_and_case_type(domain, case_type, chunk_size=100):
```

Comment:
looks like a utility function that could be used when debugging. Is there an easier way to do this?